.eslintrc.cjs

```diff
@@ -85,6 +86,7 @@ module.exports = {
     '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
     '@typescript-eslint/member-ordering': 'error',
     '@shopify/prefer-early-return': 'error',
+    'amplify-backend-rules/no-empty-catch': 'error',
```

Comment:
While this applies globally to all packages in the repo, have we considered making this rule on by default so no config is needed outside of import?

Reply:
we could do something like this https://github.com/eslint-community/eslint-plugin-promise/blob/bbcfcbf85ef97cba58f4fe4b9f46c907e04bc5a6/index.js#L28-L46 
and add to this section
https://github.com/aws-amplify/amplify-backend/blob/981bd4423e4b87217cbfef51548d2f30411cc4dc/.eslintrc.cjs#L7-L13

I'll give it a try.

Reply:
seems to be working, added.